### PR TITLE
Functions - add a slide about return statement

### PR DESCRIPTION
**Pull request by shiramax**

### CODE REVIEW

File: docs/course_materials/functions.html [hunk 1]
```diff
@@ -113,6 +113,21 @@ <h3><b>Variables Scope</b></h3>
 print(a)   # 5
                     </code></pre>
                 </section>
+
```
Comment (sleviim) [inline docs/course_materials/functions.html:116]: redundant

File: docs/course_materials/functions.html [hunk 2]
```diff
@@ -113,6 +113,21 @@ <h3><b>Variables Scope</b></h3>
...
+              <section>
+                <h3><b>return statement</b></h3>
```
Comment (sleviim) [inline docs/course_materials/functions.html:118]: Return

File: docs/course_materials/functions.html [hunk 3]
```diff
@@ -130,6 +145,7 @@ <h3><b>Summary</b></h3>
                         <li>A function can have multiple parameters</li>
                         <li>Parameter data types are not specified</li>
                         <li>Be aware of variables scope</li>
+                        <li>Use the return statement if you want that the function will return a value</li>
```
Comment (sleviim) [inline docs/course_materials/functions.html:148]: remove 'that'

File: docs/course_materials/functions.html [hunk 4]
```diff
@@ -113,6 +113,21 @@ <h3><b>Variables Scope</b></h3>
...
+                <h3><b>return statement</b></h3>
+                <ul>
+                  <li>To let a function return a value, use the return statement</li>
+                </ul>
+                <pre><code data-trim data-noescape class="hljs lang-python">
+def my_function(x):
+    return 5 * x
+
+my_function(3)
+print(my_function(5))
```
Comment (sleviim) [inline docs/course_materials/functions.html:127]: add # 25

File: docs/course_materials/functions.html [hunk 5]
```diff
@@ -113,6 +113,21 @@ <h3><b>Variables Scope</b></h3>
...
+print(my_function(9))
```
Comment (sleviim) [inline docs/course_materials/functions.html:128]: add # 45

**Review by sleviim — CHANGES_REQUESTED** [review]

File: docs/course_materials/functions.html (hunk 2 above)
Comment (shiramax) [reply docs/course_materials/functions.html:118]: but the return statement should be lower case

File: docs/course_materials/functions.html (hunk 1 above)
Comment (shiramax) [reply docs/course_materials/functions.html:116]: done

File: docs/course_materials/functions.html (hunk 3 above)
Comment (shiramax) [reply docs/course_materials/functions.html:148]: done

File: docs/course_materials/functions.html (hunk 2 above)
Comment (sleviim) [reply docs/course_materials/functions.html:117]: maybe "The return statement"?

**Pull request merged**